autofs-5.1.1 - fix missing source sss in multi map lookup

From: Ian Kent <raven@themaw.net>

The sss source isn't accounted for in the multi map lookup module.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG              |    1 +
 modules/lookup_multi.c |    3 ++-
 2 files changed, 3 insertions(+), 1 deletion(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -193,6 +193,7 @@
 - fix config old name lookup.
 - fix error handling on ldap bind fail.
 - fix gcc5 complaints.
+- fix missing source sss in multi map lookup.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/modules/lookup_multi.c
+++ autofs-5.0.7/modules/lookup_multi.c
@@ -58,7 +58,8 @@ static struct lookup_mod *nss_open_looku
 	    !strncmp(argv[0], "nisplus", 7) ||
 	    !strncmp(argv[0], "nis", 3) ||
 	    !strncmp(argv[0], "ldaps", 5) ||
-	    !strncmp(argv[0], "ldap", 4)) {
+	    !strncmp(argv[0], "ldap", 4) ||
+	    !strncmp(argv[0], "sss", 3)) {
 		const char *fmt = strchr(argv[0], ',');
 		if (fmt)
 			fmt++;
